articles/models.py

```diff
@@ -1,11 +1,15 @@
 from __future__ import absolute_import, division, unicode_literals
 
+import json
+import urllib2
```

Comment:
instead of "import urllib2" use "import requests"  It's a nicer library and should be 2/3 compatible.  Also it eliminates the need for import json